.circleci/config.yml

```diff
@@ -96,13 +96,30 @@ orbs:
               paths:
                 - << parameters.operatorDir >>/build/_output
                 - << parameters.operatorDir >>/vendor
-          - deploy:
-              name: Push image to InfluxData's Quay repo
+          - run:
+              name: Install gcloud CLI 
+              command: |
+                curl -Lo gcloud.tar.gz https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-424.0.0-linux-x86_64.tar.gz
```

Comment:
Can we use a sha256, a gpg or install this via a package manager?

Reply:
This is what Google recommends for versioned, non-interactive use cases like CO. They might offer hashes, though, that I can verify after download. Would that suffice?

Reply:
A sha256 would be fine. You would then update it whenever you change the version (it is not uncommon for upstream to not recommend sha256 in example code, but they often provide sha256s we should verify since it is a meaningful defense against certain supply chain attacks).

https://cloud.google.com/sdk/docs/install#linux lists the sha256 as `b6aba3ed40128b98ff74cafa3d26edb7bbb3530db7661aac26bd2fd1147b17b0`. You can verify this using something like https://github.com/influxdata/idpe/blob/master/scripts/docker/install-rust-compiler.sh#L26